lib/platform/github/index.ts

```diff
@@ -1636,7 +1636,27 @@ export async function getVulnerabilityAlerts(): Promise<VulnerabilityAlert[]> {
     if (vulnerabilityAlerts?.length) {
       alerts = vulnerabilityAlerts.map((edge) => edge.node);
       if (alerts.length) {
-        logger.debug({ alerts }, 'Found GitHub vulnerability alerts');
+        const importantSeverities = ['HIGH', 'MODERATE'];
+        for (const alert of alerts) {
+          if (importantSeverities.includes(alert.securityAdvisory.severity)) {
+            const {
+              package: { name, ecosystem },
+              vulnerableVersionRange,
+              firstPatchedVersion,
+            } = alert.securityVulnerability;
+            const result: Record<string, string> = {
```

Comment:
Can you strongly type this structure? This makes sure we save enough info to use later